### PR TITLE
Teleporter rework

### PR DESCRIPTION
**Pull request by Dream-Master**

### CODE REVIEW

**Comment by Dream-Master** [comment]
DRAFT not merge

**Review by Ethryan — APPROVED** [review]
Looks good

File: scripts/Thaumic-Horizons.zs [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+mods.thaumcraft.Infusion.removeRecipe(<ThaumicHorizons:recombinator>);
+
+
+
+
+// --- Adding Recipes ---
+
+
+
+// --- Planar Conduit
```
Comment (KiloJoel) [inline scripts/Thaumic-Horizons.zs:28]: These general thaumic horizons nerfs really shouldn't happen. This gates the majority of thaumic horizons stuff to tungstensteel coils (for enderium). Most of this stuff is earlygame magic things, like slightly increasing the power of an energised node, or temporarily making a hungry node safe. This also nerfs the entirety of void crafting, which didn't need a nerf.

**Comment by Dream-Master** [comment]
yes changed the recipe
image
image

**Review by KiloJoel — COMMENTED** [review]
Looks good now, with the planar conduit back to hv

**Pull request merged**